src/status_im/browser/core.cljs

```diff
@@ -209,6 +215,7 @@
         (fx/merge cofx
                   (update-browser-history browser resolved-url)
                   (handle-pdf url)
+                  (handle-message-link url)
```

Comment:
hm, do we want to show browser selection for every clicked link? maybe here better to handle it with `when` 
```
(when (utils.universal-links/universal-link? link)
    (utils.universal-links/handle-url cofx link)
```

Reply:
The check is already done in the fx implementation.

Reply:
yes and that check has `if` that means it will show browser selection each time user navigates in dapp

Reply:
so i mean to have something like 
```
(fx/defn handle-universal-link
  [cofx link]
  (when (utils.universal-links/universal-link? link)
    (utils.universal-links/handle-url cofx link)))
```

Reply:
:+1: 

`handle-universal-link` is already used elsewhere so I introduced a different one.

Reply:
ok, thanks